### PR TITLE
chore: manually build the example container image

### PR DESCRIPTION
**Pull request by ferrarimarco**
Add instructions to build the distributed TensorFlow Federated example in the example README, instead of having Terraform orchestrate the build and push process.

This simplifies the root Terraform module, by removing something that's needed only for an example, and it's also unlikely for users to have a build pipeline orchestrated this way.

This is the first of a few PRs to simplify the workload deployment for the TFF example.

/cc @grapesfrog

### CODE REVIEW

File: examples/federated-learning/tff/distributed-fl-simulation-k8s/README.md [hunk 1]
```diff
@@ -156,10 +158,38 @@ You can run this example in different runtime environments:
...
    [GKE Workloads dashboard](https://cloud.google.com/kubernetes-engine/docs/concepts/dashboards#workloads)
    in their respective GKE clusters.
 
+### Build the example container image, and push it to the container image registry
+
+1. Build the example container image locally on your host:
+
+   ```sh
+   DISTRIBUTED_TFF_EXAMPLE_CONTAINER_IMAGE_REPOSITORY_HOSTNAME="$(terraform output -raw container_image_repository_fully_qualified_hostname)"
+   DISTRIBUTED_TFF_EXAMPLE_CONTAINER_IMAGE_LOCALIZED_ID="${DISTRIBUTED_TFF_EXAMPLE_CONTAINER_IMAGE_REPOSITORY_HOSTNAME}/$(terraform output -raw container_image_repository_name)/tff-runtime:latest"
```
Comment (laurentgrangeau) [inline examples/federated-learning/tff/distributed-fl-simulation-k8s/README.md:172]: Do you think it's a good idea to tag this container as latest ? I'll tag a fixed version to have reproducibility

**Review by laurentgrangeau — COMMENTED** [review]
One small comment about tagging the image to latest

File: examples/federated-learning/tff/distributed-fl-simulation-k8s/README.md (hunk 1 above)
Comment (ferrarimarco) [reply examples/federated-learning/tff/distributed-fl-simulation-k8s/README.md:172]: We can tag it what whatever, but for an example it's probably fine. For reproducibility, you'd want to use a hash maybe.

I can use something like `0.0.1` or whatever, if you prefer :)

Comment (laurentgrangeau) [reply examples/federated-learning/tff/distributed-fl-simulation-k8s/README.md:172]: Let's use `0.0.1` for reproducibility purpose :)

**Review by laurentgrangeau — APPROVED** [review]
LGTM

**Pull request merged**